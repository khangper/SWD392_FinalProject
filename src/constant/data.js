export const API_KEY = "API_KEY";

export const FETCH_HOME_LIVESTREAM_REQUEST = "FETCH_HOME_LIVESTREAM_REQUEST";
export const FETCH_HOME_LIVESTREAM_SUCCESS = "FETCH_HOME_LIVESTREAM_SUCCESS";
export const FETCH_HOME_LIVESTREAM_FAILURE = "FETCH_HOME_LIVESTREAM_FAILURE";

export const FETCH_HOME_FEATURED_COURSES_REQUEST =
  "FETCH_HOME_FEATURED_COURSES_REQUEST";
export const FETCH_HOME_FEATURED_COURSES_SUCCESS =
  "FETCH_HOME_FEATURED_COURSES_SUCCESS";
export const FETCH_HOME_FEATURED_COURSES_FAILURE =
  "FETCH_HOME_FEATURED_COURSES_FAILURE";
// All Live Stream
export const FETCH_STREAMS_REQUEST = "FETCH_STREAMS_REQUEST";
export const FETCH_STREAMS_SUCCESS = "FETCH_STREAMS_SUCCESS";
export const FETCH_STREAMS_FAILURE = "FETCH_STREAMS_FAILURE";

//All Instructor
export const FETCH_INSTRUCTORS_REQUEST = "FETCH_INSTRUCTORS_REQUEST";
export const FETCH_INSTRUCTORS_SUCCESS = "FETCH_INSTRUCTORS_SUCCESS";
export const FETCH_INSTRUCTORS_FAILURE = "FETCH_INSTRUCTORS_FAILURE";
export const SEARCH_INSTRUCTORS_REQUEST = "SEARCH_INSTRUCTORS_REQUEST";
export const FETCH_INSTRUCTOR_BY_ID_REQUEST = "FETCH_INSTRUCTOR_BY_ID_REQUEST";

// SaveCourse.js
export const FETCH_SAVECOURSES_REQUEST = "FETCH_SAVECOURSES_REQUEST";
export const FETCH_SAVECOURSES_SUCCESS = "FETCH_SAVECOURSES_SUCCESS";
export const FETCH_SAVECOURSES_FAILURE = "FETCH_SAVECOURSES_FAILURE";
export const DELETE_SAVECOURSE_REQUEST = "DELETE_SAVECOURSE_REQUEST";
export const DELETE_SAVECOURSE_SUCCESS = "DELETE_SAVECOURSE_SUCCESS";
export const DELETE_SAVECOURSE_FAILURE = "DELETE_SAVECOURSE_FAILURE";

// Shoppingcart.js
export const ShoppingCart_API_URL =
  "https://66908916c0a7969efd9c67ed.mockapi.io/ojt-repo/ShoppingCart";
export const ADD_TO_CART = "ADD_TO_CART";
export const REMOVE_FROM_CART = "REMOVE_FROM_CART";
export const APPLY_DISCOUNT = "APPLY_DISCOUNT";
export const ADD_TO_CART_FAILURE = " ADD_TO_CART_FAILURE";
export const ADD_TO_CART_SUCCESS = "APPLY_DISCOUNT";

//MyCourse_Course
export const FETCH_COURSES_REQUEST = "FETCH_COURSES_REQUEST";
export const FETCH_COURSES_SUCCESS = "FETCH_COURSES_SUCCESS";
export const FETCH_COURSES_FAILURE = "FETCH_COURSES_FAILURE";

export const EDIT_COURSE_REQUEST = "EDIT_COURSE_REQUEST";
export const EDIT_COURSE_SUCCESS = "EDIT_COURSE_SUCCESS";
export const EDIT_COURSE_FAILURE = "EDIT_COURSE_FAILURE";

export const DELETE_COURSE_REQUEST = "DELETE_COURSE_REQUEST";
export const DELETE_COURSE_SUCCESS = "DELETE_COURSE_SUCCESS";
export const DELETE_COURSE_FAILURE = "DELETE_COURSE_FAILURE";

//UpcommingCourse_Course
export const FETCH_UPCOMMINGCOURSES_REQUEST = "FETCH_UPCOMMINGCOURSES_REQUEST";
export const FETCH_UPCOMMINGCOURSES_SUCCESS = "FETCH_UPCOMMINGCOURSES_SUCCESS";
export const FETCH_UPCOMMINGCOURSES_FAILURE = "FETCH_UPCOMMINGCOURSES_FAILURE";

export const EDIT_UPCOMMINGCOURSE_REQUEST = "EDIT_UPCOMMINGCOURSE_REQUEST";
export const EDIT_UPCOMMINGCOURSE_SUCCESS = "EDIT_UPCOMMINGCOURSE_SUCCESS";
export const EDIT_UPCOMMINGCOURSE_FAILURE = "EDIT_UPCOMMINGCOURSE_FAILURE";

export const DELETE_UPCOMMINGCOURSE_REQUEST = "DELETE_UPCOMMINGCOURSE_REQUEST";
export const DELETE_UPCOMMINGCOURSE_SUCCESS = "DELETE_UPCOMMINGCOURSE_SUCCESS";
export const DELETE_UPCOMMINGCOURSE_FAILURE = "DELETE_UPCOMMINGCOURSE_FAILURE";
