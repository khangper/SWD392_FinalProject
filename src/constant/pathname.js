export const PATH_NAME = {
  HOME: "/home",
  EXPLORE: "/explore",
  LIVE_OUTPUT: "/live_output/:id",
  ADD_LIVE_STREAM: "/add_livestream",
  ALL_LIVE_STREAM: "/all_livestream",
  ALL_INSTRUCTOR_LIST: "/allinstructorlist/:id",
  HELP: "/help",
  REPORT_HISTORY: "/reporthistory",
  SEND_FEEDBACK: "/sendfeedback",
  OTHER_INSTRUCTOR_VIEW: "/other-instructor-view/:id",
  MY_INSTRUCTOR_PROFILE: "/my-instructor-profile",
  SAVE_COURSES: "/save-courses",
  SETTING: "/setting",
  MY_COURSE: "/courses",
  LOGIN: "/login",
  FORGOT_PASSWORD: "/forgot-password",
  SIGN_UP: "/sign-up",
  SIGN_UP_STEP: "/SignUpStep",
  THANKS: "/thanks",
  ERROR_404: "/error404",
  INVOICE: "/invoice",
  PRESS: "/press",
  SEARCH_RESULT: "/search_result",
  PAID_MEMBERSHIP: "/paidmembership",
  ABOUT_US: "/about-us",
  OUR_BLOG: "/our-blog",
  BLOG_SINGLE_VIEW: "/blog-single-view",
  CHECKOUT: "/checkout",
  SHOPPING_CART: "/shoppingcart",
  HELP_VIEW_1: "/helpview1",
  HELP_VIEW_2: "/helpview2",
  COMPANY_DETAILS: "/company-details",
  CAREER: "/career",
  APPLY_JOB: "/apply-job",
  CONTACT_US: "/contactus",
  TERMS_OF_USE: "/termsofuse",
  CERTIFICATION_FILL_FORM: "/certificationfillform",
  CERTIFICATION_TEST: "/certificates-test",
  TEST_RESULT: "/test-result",
  MESSAGE: "/messages",
  COURSES_DETAIL_VIEW: "/courses-detail-view/:id",
  PREVIEWS: "/reviews",
  COMING_SOON: "/coming-soon",
  INS_DASHBOARD: "/ins-dashboard",
  PURCHASE_COURSE: "/PurchaseCourse",
  NOTIFICATON: "/notifications",
  DOWLOADCOURSE: "/dowloadcourse",
  CERTIFICATION_CENTER: "/certificationcenter",
  SITEMAP: "/sitemap",
  EARNING: "/earning",
  PAYOUT: "/payout",
  VERTIFICATION: "/vertification",
  MYCERTIFICATIONS: "/mycertifications",
  CREDIT: "/credits",
  STATEMENTIS: "/statementis",
  STATEMENTST: "/statementst",
  ANALYTICS: "/analytics",
  CREATE_NEW_COURSE: "/create-course",
  STD_DASHBOARD: "/student-dashboard",
  NOTIFICATIONSTUDENT: "/notificationsstudent",
};
