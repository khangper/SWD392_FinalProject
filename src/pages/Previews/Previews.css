.p-star {
    width: 20px;
    top: 1px;
    position: relative;
    margin-right: 2px;
}

.p-progress {
    width: 30%;
    height: 10px;
    background-color: #e0e0e0;
    margin-right: 10px;
}

.p-rating-box {
    display: flex;
    align-items: center;
    width: 50%;
}

.p-rate002 {
    width: 2%;
    margin-left: 10px;
    font-size: 1em;
}