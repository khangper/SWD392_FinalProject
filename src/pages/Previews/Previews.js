import React from 'react'
import './Previews.css'
import star from "../../assets/star.png";
import ratingStar from "../../assets/rating.png"
import search from "../../assets/search.png";

const Previews = () => {
    return (
        <div>
            <div className="tab-pane fade active show" >
                <div className="student_reviews">
                    <div className="courses_detail_view-row">
                        <div className="cdv-col-lg-5">
                            <div className="reviews_left">
                                <h3>Student Feedback</h3>
                                <div className="total_rating">
                                    <div className="_rate001">4.6</div>
                                    <div className="p-rating-box">
                                        <img src={star} className='cdv-rate-star' />
                                        <img src={star} className='cdv-rate-star' />
                                        <img src={star} className='cdv-rate-star' />
                                        <img src={star} className='cdv-rate-star' />
                                        <img className='p-star' src={ratingStar}></img>
                                    </div>
                                    <div className="p-rate002">Course Rating</div>
                                </div>
                                <div className="_rate003">
                                    <div className="_rate004">
                                        <div className="p-progress">
                                            <div className="progress-bar w-70"></div>
                                        </div>
                                        <div className="p-rating-box">
                                            <img src={star} className='cdv-rate-star' />
                                            <img src={star} className='cdv-rate-star' />
                                            <img src={star} className='cdv-rate-star' />
                                            <img src={star} className='cdv-rate-star' />
                                            <img src={star} className='cdv-rate-star' />
                                        </div>
                                        <div className="p-rate002">70%</div>
                                    </div>
                                    <div className="_rate004">
                                        <div className="p-progress">
                                            <div className="progress-bar w-30"></div>
                                        </div>
                                        <div className="p-rating-box">
                                            <img src={star} className='cdv-rate-star' />
                                            <img src={star} className='cdv-rate-star' />
                                            <img src={star} className='cdv-rate-star' />
                                            <img src={star} className='cdv-rate-star' />
                                            <img className='p-star' src={ratingStar}></img>
                                        </div>
                                        <div className="p-rate002">40%</div>
                                    </div>
                                    <div className="_rate004">
                                        <div className="p-progress">
                                            <div className="progress-bar w-5"></div>
                                        </div>
                                        <div className="p-rating-box">
                                            <img src={star} className='cdv-rate-star' />
                                            <img src={star} className='cdv-rate-star' />
                                            <img src={star} className='cdv-rate-star' />
                                            <img className='p-star' src={ratingStar}></img>
                                            <img className='p-star' src={ratingStar}></img>
                                        </div>
                                        <div className="p-rate002">5%</div>
                                    </div>
                                    <div className="_rate004">
                                        <div className="p-progress">
                                            <div className="progress-bar w-2"></div>
                                        </div>
                                        <div className="p-rating-box">
                                            <img src={star} className='cdv-rate-star' />
                                            <img src={star} className='cdv-rate-star' />
                                            <img className='p-star' src={ratingStar}></img>
                                            <img className='p-star' src={ratingStar}></img>
                                            <img className='p-star' src={ratingStar}></img>
                                        </div>
                                        <div className="p-rate002">1%</div>
                                    </div>
                                    <div className="_rate004">
                                        <div className="p-progress">
                                            <div className="progress-bar w-1"></div>
                                        </div>
                                        <div className="p-rating-box">
                                            <img src={star} className='cdv-rate-star' />
                                            <img className='p-star' src={ratingStar}></img>
                                            <img className='p-star' src={ratingStar}></img>
                                            <img className='p-star' src={ratingStar}></img>
                                            <img className='p-star' src={ratingStar}></img>
                                        </div>
                                        <div className="p-rate002">1%</div>
                                    </div>
                                </div>
                            </div>
                        </div>
                        <div className="cdv-col-lg-7">
                            <div className="review_right">
                                <div className="review_right_heading">
                                    <h3>Reviews</h3>
                                    <div className="review_search">
                                        <input className="rv_srch" type="text" placeholder="Search reviews..." />
                                        <button className="rvsrch_btn"><img src={search} /></button>
                                    </div>
                                </div>
                            </div>
                            <div className="review_all120">
                                <div className="review_item">
                                    <div className="review_usr_dt">
                                        <div className="oiv-img160">
                                            <img src="https://gambolthemes.net/html-items/cursus-new-demo/images/left-imgs/img-1.jpg" alt="" />
                                        </div>
                                        <div className="oiv-cmnt_name">
                                            <h4 className="oiv-tutor_name">John Doe</h4>
                                            <span className="oiv-cmnt_time">2 hour ago</span>
                                        </div>
                                    </div>
                                    <div className="p-rating-box ">
                                        <img src={star} className='cdv-rate-star' />
                                        <img src={star} className='cdv-rate-star' />
                                        <img src={star} className='cdv-rate-star' />
                                        <img src={star} className='cdv-rate-star' />
                                        <img className='p-star' src={ratingStar}></img>
                                    </div>
                                    <p className="cdv-cmnt_text_message">Nam gravida elit a velit rutrum, eget dapibus ex elementum. Interdum et malesuada fames ac ante ipsum primis in faucibus. Fusce lacinia, nunc sit amet tincidunt venenatis.</p>
                                    <div className="cdv-check-helpful">
                                        <span>Was this review helpful?</span>
                                        <div className="radio--group-inline-container">
                                            <div className="radio-item">
                                                <input id="radio-1" name="radio" type="radio" />
                                                <label htmlFor="radio-1" className="radio-label">Yes</label>
                                            </div>
                                            <div className="radio-item">
                                                <input id="radio-2" name="radio" type="radio" />
                                                <label htmlFor="radio-2" className="radio-label">No</label>
                                            </div>
                                        </div>
                                        <a href="#" className="report145">Report</a>
                                    </div>
                                </div>
                                <div className="review_item">
                                    <div className="review_usr_dt">
                                        <div className="oiv-img160">
                                            <img src="https://gambolthemes.net/html-items/cursus-new-demo/images/left-imgs/img-1.jpg" alt="" />
                                        </div>
                                        <div className="oiv-cmnt_name">
                                            <h4 className="oiv-tutor_name">John Doe</h4>
                                            <span className="oiv-cmnt_time">2 hour ago</span>
                                        </div>
                                    </div>
                                    <div className="p-rating-box ">
                                        <img src={star} className='cdv-rate-star' />
                                        <img src={star} className='cdv-rate-star' />
                                        <img src={star} className='cdv-rate-star' />
                                        <img src={star} className='cdv-rate-star' />
                                        <img className='p-star' src={ratingStar}></img>
                                    </div>
                                    <p className="cdv-cmnt_text_message">Nam gravida elit a velit rutrum, eget dapibus ex elementum. Interdum et malesuada fames ac ante ipsum primis in faucibus. Fusce lacinia, nunc sit amet tincidunt venenatis.</p>
                                    <div className="cdv-check-helpful">
                                        <span>Was this review helpful?</span>
                                        <div className="radio--group-inline-container">
                                            <div className="radio-item">
                                                <input id="radio-1" name="radio" type="radio" />
                                                <label htmlFor="radio-1" className="radio-label">Yes</label>
                                            </div>
                                            <div className="radio-item">
                                                <input id="radio-2" name="radio" type="radio" />
                                                <label htmlFor="radio-2" className="radio-label">No</label>
                                            </div>
                                        </div>
                                        <a href="#" className="report145">Report</a>
                                    </div>
                                </div>
                                <div className="review_item">
                                    <div className="review_usr_dt">
                                        <div className="oiv-img160">
                                            <img src="https://gambolthemes.net/html-items/cursus-new-demo/images/left-imgs/img-1.jpg" alt="" />
                                        </div>
                                        <div className="oiv-cmnt_name">
                                            <h4 className="oiv-tutor_name">John Doe</h4>
                                            <span className="oiv-cmnt_time">2 hour ago</span>
                                        </div>
                                    </div>
                                    <div className="p-rating-box ">
                                        <img src={star} className='cdv-rate-star' />
                                        <img src={star} className='cdv-rate-star' />
                                        <img src={star} className='cdv-rate-star' />
                                        <img src={star} className='cdv-rate-star' />
                                        <img className='p-star' src={ratingStar}></img>
                                    </div>
                                    <p className="cdv-cmnt_text_message">Nam gravida elit a velit rutrum, eget dapibus ex elementum. Interdum et malesuada fames ac ante ipsum primis in faucibus. Fusce lacinia, nunc sit amet tincidunt venenatis.</p>
                                    <div className="cdv-check-helpful">
                                        <span>Was this review helpful?</span>
                                        <div className="radio--group-inline-container">
                                            <div className="radio-item">
                                                <input id="radio-1" name="radio" type="radio" />
                                                <label htmlFor="radio-1" className="radio-label">Yes</label>
                                            </div>
                                            <div className="radio-item">
                                                <input id="radio-2" name="radio" type="radio" />
                                                <label htmlFor="radio-2" className="radio-label">No</label>
                                            </div>
                                        </div>
                                        <a href="#" className="report145">Report</a>
                                    </div>
                                </div>
                                <div className="review_item">
                                    <div className="review_usr_dt">
                                        <div className="oiv-img160">
                                            <img src="https://gambolthemes.net/html-items/cursus-new-demo/images/left-imgs/img-1.jpg" alt="" />
                                        </div>
                                        <div className="oiv-cmnt_name">
                                            <h4 className="oiv-tutor_name">John Doe</h4>
                                            <span className="oiv-cmnt_time">2 hour ago</span>
                                        </div>
                                    </div>
                                    <div className="p-rating-box ">
                                        <img src={star} className='cdv-rate-star' />
                                        <img src={star} className='cdv-rate-star' />
                                        <img src={star} className='cdv-rate-star' />
                                        <img src={star} className='cdv-rate-star' />
                                        <img className='p-star' src={ratingStar}></img>
                                    </div>
                                    <p className="cdv-cmnt_text_message">Nam gravida elit a velit rutrum, eget dapibus ex elementum. Interdum et malesuada fames ac ante ipsum primis in faucibus. Fusce lacinia, nunc sit amet tincidunt venenatis.</p>
                                    <div className="cdv-check-helpful">
                                        <span>Was this review helpful?</span>
                                        <div className="radio--group-inline-container">
                                            <div className="radio-item">
                                                <input id="radio-1" name="radio" type="radio" />
                                                <label htmlFor="radio-1" className="radio-label">Yes</label>
                                            </div>
                                            <div className="radio-item">
                                                <input id="radio-2" name="radio" type="radio" />
                                                <label htmlFor="radio-2" className="radio-label">No</label>
                                            </div>
                                        </div>
                                        <a href="#" className="report145">Report</a>
                                    </div>
                                </div>
                                <div className="review_item">
                                    <div className="review_usr_dt">
                                        <div className="oiv-img160">
                                            <img src="https://gambolthemes.net/html-items/cursus-new-demo/images/left-imgs/img-1.jpg" alt="" />
                                        </div>
                                        <div className="oiv-cmnt_name">
                                            <h4 className="oiv-tutor_name">John Doe</h4>
                                            <span className="oiv-cmnt_time">2 hour ago</span>
                                        </div>
                                    </div>
                                    <div className="p-rating-box ">
                                        <img src={star} className='cdv-rate-star' />
                                        <img src={star} className='cdv-rate-star' />
                                        <img src={star} className='cdv-rate-star' />
                                        <img src={star} className='cdv-rate-star' />
                                        <img className='p-star' src={ratingStar}></img>
                                    </div>
                                    <p className="cdv-cmnt_text_message">Nam gravida elit a velit rutrum, eget dapibus ex elementum. Interdum et malesuada fames ac ante ipsum primis in faucibus. Fusce lacinia, nunc sit amet tincidunt venenatis.</p>
                                    <div className="cdv-check-helpful">
                                        <span>Was this review helpful?</span>
                                        <div className="radio--group-inline-container">
                                            <div className="radio-item">
                                                <input id="radio-1" name="radio" type="radio" />
                                                <label htmlFor="radio-1" className="radio-label">Yes</label>
                                            </div>
                                            <div className="radio-item">
                                                <input id="radio-2" name="radio" type="radio" />
                                                <label htmlFor="radio-2" className="radio-label">No</label>
                                            </div>
                                        </div>
                                        <a href="#" className="report145">Report</a>
                                    </div>
                                </div>
                                <div className="review_item">
                                    <a href="#" className="more_reviews">See More Reviews</a>
                                </div>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    )
}

export default Previews