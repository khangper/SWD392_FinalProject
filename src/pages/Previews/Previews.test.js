import React from "react";
import { renderWithProviders } from "../../constant/test-util";
import Previews from "./Previews";
import { screen } from "@testing-library/react";

describe("Previews", () => {
    beforeAll(() => {
        jest.spyOn(console, "error").mockImplementation(() => { });
        jest.spyOn(console, "log").mockImplementation(() => { });
    });

    afterAll(() => {
        console.error.mockRestore();
        console.log.mockRestore();
    });
    it("renders the Previews component", async () => {
        const { container } = renderWithProviders(<Previews />);
        expect(container).toBeDefined();
        expect(container).toMatchSnapshot();
    });
})