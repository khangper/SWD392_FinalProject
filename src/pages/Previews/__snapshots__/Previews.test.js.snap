// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Previews renders the Previews component 1`] = `
<div>
  <div>
    <div
      class="tab-pane fade active show"
    >
      <div
        class="student_reviews"
      >
        <div
          class="courses_detail_view-row"
        >
          <div
            class="cdv-col-lg-5"
          >
            <div
              class="reviews_left"
            >
              <h3>
                Student Feedback
              </h3>
              <div
                class="total_rating"
              >
                <div
                  class="_rate001"
                >
                  4.0
                </div>
                <div
                  class="p-rating-box"
                >
                  <img
                    class="cdv-rate-star"
                    src="star.png"
                  />
                  <img
                    class="cdv-rate-star"
                    src="star.png"
                  />
                  <img
                    class="cdv-rate-star"
                    src="star.png"
                  />
                  <img
                    class="cdv-rate-star"
                    src="star.png"
                  />
                  <img
                    class="p-star"
                    src="rating.png"
                  />
                </div>
                <div
                  class="p-rate002"
                >
                  Course Rating
                </div>
              </div>
              <div
                class="_rate003"
              >
                <div
                  class="_rate004"
                >
                  <div
                    class="p-progress"
                  >
                    <div
                      class="p-progress-bar"
                    />
                  </div>
                  <div
                    class="p-rating-box"
                  >
                    <img
                      class="cdv-rate-star"
                      src="star.png"
                    />
                    <img
                      class="cdv-rate-star"
                      src="star.png"
                    />
                    <img
                      class="cdv-rate-star"
                      src="star.png"
                    />
                    <img
                      class="cdv-rate-star"
                      src="star.png"
                    />
                    <img
                      class="cdv-rate-star"
                      src="star.png"
                    />
                  </div>
                  <div
                    class="p-rate002"
                  >
                    100%
                  </div>
                </div>
                <div
                  class="_rate004"
                >
                  <div
                    class="p-progress"
                  >
                    <div
                      class="p-progress-bar1"
                    />
                  </div>
                  <div
                    class="p-rating-box"
                  >
                    <img
                      class="cdv-rate-star"
                      src="star.png"
                    />
                    <img
                      class="cdv-rate-star"
                      src="star.png"
                    />
                    <img
                      class="cdv-rate-star"
                      src="star.png"
                    />
                    <img
                      class="cdv-rate-star"
                      src="star.png"
                    />
                    <img
                      class="p-star"
                      src="rating.png"
                    />
                  </div>
                  <div
                    class="p-rate002"
                  >
                    80%
                  </div>
                </div>
                <div
                  class="_rate004"
                >
                  <div
                    class="p-progress"
                  >
                    <div
                      class="p-progress-bar2"
                    />
                  </div>
                  <div
                    class="p-rating-box"
                  >
                    <img
                      class="cdv-rate-star"
                      src="star.png"
                    />
                    <img
                      class="cdv-rate-star"
                      src="star.png"
                    />
                    <img
                      class="cdv-rate-star"
                      src="star.png"
                    />
                    <img
                      class="p-star"
                      src="rating.png"
                    />
                    <img
                      class="p-star"
                      src="rating.png"
                    />
                  </div>
                  <div
                    class="p-rate002"
                  >
                    60%
                  </div>
                </div>
                <div
                  class="_rate004"
                >
                  <div
                    class="p-progress"
                  >
                    <div
                      class="p-progress-bar3"
                    />
                  </div>
                  <div
                    class="p-rating-box"
                  >
                    <img
                      class="cdv-rate-star"
                      src="star.png"
                    />
                    <img
                      class="cdv-rate-star"
                      src="star.png"
                    />
                    <img
                      class="p-star"
                      src="rating.png"
                    />
                    <img
                      class="p-star"
                      src="rating.png"
                    />
                    <img
                      class="p-star"
                      src="rating.png"
                    />
                  </div>
                  <div
                    class="p-rate002"
                  >
                    40%
                  </div>
                </div>
                <div
                  class="_rate004"
                >
                  <div
                    class="p-progress"
                  >
                    <div
                      class="p-progress-bar4"
                    />
                  </div>
                  <div
                    class="p-rating-box"
                  >
                    <img
                      class="cdv-rate-star"
                      src="star.png"
                    />
                    <img
                      class="p-star"
                      src="rating.png"
                    />
                    <img
                      class="p-star"
                      src="rating.png"
                    />
                    <img
                      class="p-star"
                      src="rating.png"
                    />
                    <img
                      class="p-star"
                      src="rating.png"
                    />
                  </div>
                  <div
                    class="p-rate002"
                  >
                    20%
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="cdv-col-lg-7"
          >
            <div
              class="review_right"
            >
              <div
                class="review_right_heading"
              >
                <h3>
                  Reviews
                </h3>
                <div
                  class="review_search"
                >
                  <input
                    class="rv_srch"
                    placeholder="Search reviews..."
                    type="text"
                  />
                  <button
                    class="rvsrch_btn"
                  >
                    <img
                      src="search.png"
                    />
                  </button>
                </div>
              </div>
            </div>
            <div
              class="review_all120"
            >
              <div
                class="review_item"
              >
                <a
                  class="more_reviews"
                  href="#"
                >
                  See More Reviews
                </a>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
